[docs] #159, #171, #179: Update config sections (#213)
hyperledger/iroha-2-docs: src/guide/advanced/running-iroha-on-bare-metal.md

variables can offer a better user experience if done right, but is more
error-prone, particularly for exotic systems (Windows).

::: info

For this chapter, we assume you have learned about
[configuration and management](./../configure/intro.md) in Iroha 2. Here we

> offer you instructions for running Iroha on bare metal without going into

Reviewer: ```suggestion
offer you instructions to run Iroha on bare metal without going into
```

> details about various configuration options available to you.

Reviewer: ```suggestion
details about various configuration options available.
```

You can always check
[sample configuration files](./../configure/sample-configuration.md) for
`/configs/peer/genesis.json` and `/configs/peer/config.json`, or refer to
[peer configuration options](./../configure/peer-configuration.md) for more
details.

> You will find the complete list of options in

Reviewer: ```suggestion
The complete list of options is available in the
```

    ```

3.  Register your peer to a network, and make sure to add at least four of
    the peers on that network to the
    [`TRUSTED_PEERS`](./../configure/peer-configuration.md#trusted-peers)
    array in your configuration file.

4.  Determine the web socket that the other peers will use to connect to
    you. Make sure that the port is open and use that address
    ([`P2P_ADDR`](./../configure/peer-configuration.md#p2p_addr)) in your
    `/configs/peer/config.json`.

> 5.  After you finished editing the configuration file, deploy Iroha by

Reviewer: ```suggestion
5.  After you have finished editing the configuration file, deploy Iroha by
```
